images-list

```diff
@@ -788,7 +792,7 @@ quay.io/calico/node rancher/mirrored-calico-node v3.22.2
 quay.io/calico/node rancher/mirrored-calico-node v3.23.0
 quay.io/calico/node rancher/mirrored-calico-node v3.23.1
 quay.io/calico/node rancher/mirrored-calico-node v3.23.3
-quay.io/calico/node rancher/mirrored-calico-node v3.24.1
```

Comment:
you don't have to remove an existing image.

Reply:
indeed, copy/paste mistake